composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
@e0ipso not really as I was only able to build this because @cjgratacos had created a branch named https://github.com/contentacms/contenta_jsonapi/tree/test-drupal-8.4.x which itself has the composer change pointing to `"drupal/core": "8.4.x-dev"` 

But if you can point me in a direction on how to do it than I can test it, document it and wrap this up.